compiler/pash_ptempfile_name.sh

```diff
@@ -1,3 +1,22 @@
 #!/usr/bin/env bash
+if type lsb_release >/dev/null 2>&1 ; then
+    distro=$(lsb_release -i -s)
+elif [ -e /etc/os-release ] ; then
+    distro=$(awk -F= '$1 == "ID" {print $2}' /etc/os-release)
+fi
 
-echo "$(mktemp --tmpdir="$PASH_TMP_PREFIX" -u pash_XXXXXXXXXX)"
\ No newline at end of file
+# convert to lowercase
+distro=$(printf '%s\n' "$distro" | LC_ALL=C tr '[:upper:]' '[:lower:]')
+# now do different things depending on distro
+case "$distro" in
+    freebsd*)  
+        # bsd is so fun :)
```

Comment:
:')

Reply:
bigrams and alt-bigrams hang on their execution, but I am pretty positive that this is a bash related issue rather than pash. We are going to investigate it with @nvasilakis . All the other tests pass!

Reply:
great, thanks :)